pyro/contrib/autoguide/__init__.py

```diff
@@ -539,7 +552,7 @@ def __call__(self, *args, **kwargs):
         for site, Dist, param_spec in self._discrete_sites:
             name = site["name"]
             dist_params = {
-                param_name: pyro.param("auto_discrete_{}_{}".format(name, param_name), param_init,
+                param_name: pyro.param("{}_{}_{}".format(self.name, name, param_name), param_init,
```

Comment:
`auto_discrete` will now default to `auto`; is that intended

Reply:
Yes, that is intended.